examples/quiet.rs

```diff
@@ -5,7 +5,6 @@ use tracing_tree::HierarchicalLayer;
 fn main() {
     let layer = HierarchicalLayer::default()
         .with_writer(std::io::stdout)
-        .with_indent_lines(true)
```

Comment:
Please keep this in. If you want to test without indent lines, add a separate test